src/tuner/nccl_ofi_model.c

```diff
@@ -34,68 +82,55 @@ float nccl_ofi_tuner_compute_cost(struct nccl_ofi_tuner_model_params *params, st
 	 * device. Costs associated with out-of-order completions that could
 	 * stall the pipeline should be captured here as well.
 	 */
-	const float base_latency = (*base_costs)[func][algo][proto];
-	const float net_lat = (proto == NCCL_PROTO_SIMPLE)
-		? params->net_lat + ofi_nccl_tuner_net_comp_overhead()
-		: params->net_lat;
-	const float p2p_lat = nccl_nvlink_lat[algo][proto];
-
 	switch(func) {
 	case ncclFuncAllReduce:
 		switch(algo) {
 		case NCCL_ALGO_RING:
-			num_steps = 2 * (dims->num_ranks - 1);
-			num_internode_steps = 2 * dims->num_nodes;
-			latency = (num_internode_steps * net_lat)
-				  + (num_steps - num_internode_steps) * p2p_lat;
-			bw = params->internode_bw * params->num_rails * ofi_nccl_tuner_num_channels();
-			break;
-
+		{
+			const int num_steps = 2 * (dims->num_ranks - 1);
+			const int num_internode_steps = 2 * dims->num_nodes;
+			return (num_internode_steps * net_latency + (num_steps - num_internode_steps) * p2p_latency + base_latency) * pipe_ops;
+		}
 		case NCCL_ALGO_NVLS_TREE:
-			latency = 2 * (p2p_lat + (log2(dims->num_nodes) * net_lat));
-			bw = NCCL_OFI_MIN(params->intranode_bw, (params->internode_bw * params->num_rails) / 2)
-			     * ofi_nccl_tuner_num_channels();
-			break;
-
+			return ((p2p_latency + net_latency * log2(dims->num_nodes)) * 2 + base_latency) * pipe_ops;
```

Comment:
So we don't get confused again, we should capture in a comment that we are attributing the NVL startup costs to both trips up and down the tree.